### ISSUE TITLE
all '-->' replaced with */ in jsx node

### ISSUE THREAD

**Issue by jpambrun** [mdx-js/mdx#256]
I am very new to this. I am trying to use mermaid to draw graphs in mdx.
The minimum working example for mermaid is:
```
graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
```
Creating a jsx component following [this](https://www.developmentsindigital.com/posts/2017-10-26-mermaid-in-react/), it should translate to:
``` jsx
<Mermaid id='someid' content={`graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
`}/>
```

However, this wont work because all `-->` are somewhat blindly translated to `*/` [here](https://github.com/mdx-js/mdx/blob/e883828207f138f06f54522a5ff7a4f8337834dd/packages/mdx/mdx-hast-to-jsx.js#L33).

Is there any workaround?

**Comment by silvenon** [comment]
Sorry about that, it's my naive implementation of comments. I'll try to fix this.

**Comment by silvenon** [comment]
Resolved with #260.
